README.md

```diff
@@ -1,3 +1,44 @@
-# IDR metadata
+# IDR studies
 
 [![Published screens](https://img.shields.io/badge/dynamic/json.svg?label=Published%20&url=http%3A%2F%2Fidr.openmicroscopy.org%2Fapi%2Fv0%2Fm%2Fscreens%2F%3Flimit%3D0&query=meta.totalCount&colorB=blue&suffix=%20high%20content%20screens)](https://idr.openmicroscopy.org) [![Published experiments](https://img.shields.io/badge/dynamic/json.svg?label=Published%20&url=http%3A%2F%2Fidr.openmicroscopy.org%2Fapi%2Fv0%2Fm%2Fprojects%2F%3Flimit%3D0&query=meta.totalCount&colorB=blue&suffix=%20experiments)](https://idr.openmicroscopy.org)
+
+All metadata associated with published studies in IDR is managed.
+
+## Study name 
+
+After acceptance, IDR studies must be named as `idr<NNNN>-<name>-<description>`
+where `idr<NNNN>` is the accession number of the study using an incremental
+four digits integer, `<name>` is the name of one of the authors associated
+with the publication, usually the first author, and `<description>` is a short
+description of the study. The name should be lowercase.
+
+## Study repository 
+
+For each new IDR study, a repository must be created on GitHub under the [IDR](http://github.com/IDR/) organization using the study name as defined above. When ready for publication in the IDR, the study repository must be
+registered in the top-level idr-metadata repository.
+
+Study repository contain all curated metadata files associated with a study.
+The structure of each study repository should use the following layout:
```

Comment:
Link to https://github.com/IDR/idr0000-lastname-example ?

Reply:
I debated referring to https://github.com/IDR/idr0000-lastname-example since this repository only contains template files for submission but is not an example of a curated repository. I will try and add a first paragraph about submission referring to it before discussing curated metadata repository.